### DISCUSSION TITLE
Luogo nascita  - luogo filiazione

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#1068] [Supporto servizi cooperativi] [answered]
Salve,
In alcuni casi d'uso, è obbligatorio inserire il dato relativo a evento.datiDiNascita.luogoFiliazione.
Ho notato nella webapp che il campo "luogo di nascita" può contenere qualsiasi carattere. Potreste spiegarmi meglio cosa si intende esattamente per "luogo di nascita"?

**Comment by asqueglia** [comment, marked as answer]
Buongiorno, il luogo di nascita è il luogo dove è avvenuto il parto come ad esempio 'Ospedale XXX'. Chiudiamo pertanto la discussion; in caso di ulteriori richieste si prega di aprirne una altra dedicata.
